index.md

```diff
@@ -19,13 +19,28 @@ Data Carpentry's aim is to teach researchers basic concepts, skills, and tools f
 {: .prereq}
 
 > ## Data
->
-> The data for this workshop are is the [FIXME](FIXME) available on FigShare, with a CC-BY license available for reuse.
->
-> FIXME add a short description about the dataset.
->
+> Data for this lesson comes from the National Ecological Observatory Network. These data files represent teaching version of the data, 
+> with sufficient complexity to teach many aspects of  data analysis and management, but with many complexities removed to allow students
+> to focus on the core ideas and skills being taught.  
+> 
+> There are four data sets included, all of which are available 
+> [on Figshare](https://figshare.com/articles/Spatio_temporal_Series_Teaching_Data_Subsets/2009586)
+> under a CC-BY license. You can download all of the data used in this workshop by clicking 
+> [this download link](https://ndownloader.figshare.com/articles/2009586/versions/10). 
+> Clicking the download link will automatically download all of the files to your default download directory as a single compressed
+> (`.zip`) file. To expand this file, double click the folder icon in your file navigator application (for Macs, this is the Finder 
+> application).
+> 
```

Comment:
In case you want it:

The data and lessons in this workshop were originally developed through a hackathon supported by the National Ecological Observatory Network (NEON) - an NSF funded observatory in Boulder, Colorado. NEON is collecting data for 30 years to help scientists understand how our aquatic and terrestrial ecosystems are changing. The data used in these lessons cover two NEON field sites:
1. Harvard Forest (HARV) - Massachusetts, USA
2. San Joachin Experimental Range (SJER) - California, USA

Reply:
Thanks @lwasser. I added this.